include/courtier/GAsioConsumerT.hpp

```diff
@@ -1191,9 +1193,309 @@ class GAsioConsumerT
 	 //-------------------------------------------------------------------------
 };
 
-/******************************************************************************/
-////////////////////////////////////////////////////////////////////////////////
-/******************************************************************************/
+
+// Add Just anOther consumer
+ 
+template<typename processable_type>
+class GAsioConsumerPT
```

Comment:
There is essentially no documentation here. What is this class for, what differentiates it from the others, etc. Also, what is the rationale behind the "P" in the name? The T in Geneva convention stands for "Template".